src/Playwright/build/Microsoft.Playwright.targets

```diff
@@ -3,14 +3,18 @@
   <Target Name="CopyPlaywrightFilesToOutput" AfterTargets="BeforeBuild">
     <PropertyGroup>
       <PlaywrightPlatform Condition="'$(PlaywrightPlatform)' == ''">%(RuntimePack.RuntimeIdentifier)</PlaywrightPlatform>
-      <PlaywrightPlatform Condition="'$(PlaywrightPlatform)' == '' AND $([MSBuild]::IsOSPlatform('Linux'))">linux</PlaywrightPlatform>
+      <PlaywrightPlatform Condition="'$(PlaywrightPlatform)' == '' AND $([MSBuild]::IsOSPlatform('Linux')) AND '$([System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture)' == 'X64'">linux</PlaywrightPlatform>
+      <PlaywrightPlatform Condition="'$(PlaywrightPlatform)' == '' AND $([MSBuild]::IsOSPlatform('Linux')) AND '$([System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture)' == 'Arm64'">linux-arm64</PlaywrightPlatform>
       <PlaywrightPlatform Condition="'$(PlaywrightPlatform)' == '' AND $([MSBuild]::IsOSPlatform('Windows'))">win</PlaywrightPlatform>
       <PlaywrightPlatform Condition="'$(PlaywrightPlatform)' == '' AND $([MSBuild]::IsOSPlatform('OSX'))">osx</PlaywrightPlatform>
     </PropertyGroup>
     <ItemGroup Condition="'$(PlaywrightPlatform)' != 'none'">
       <_PlaywrightCopyItems Include="$(MSBuildThisFileDirectory)..\.playwright\node\linux\**" Condition="$(PlaywrightPlatform.Contains('linux'))">
```

Comment:
`linux-arm64` includes `linux` so on linux it there will be both folders.

Reply:
yup, not sure if there is a good way around that without breaking people. (Very good catch, actually wanted to add a comment for it in this minute)

Reply:
I'm considering renaming linux to linux-amd64, not sure how many people it would break and if semver would allow it. Probably not, but the feature with specifying the arch is pretty new and most of the people probably "let the driver version autoselect".

Reply:
Note that dotnet / msbuild uses `linux-x64` instead: https://docs.microsoft.com/dotnet/core/rid-catalog#linux-rids

Reply:
yeah I'd vote to use these rids instead of our custom values which we added a few weeks ago (linux, mac, windows).

Reply:
We added `linux` / `win` / `osx` which are valid dotnet / msbuild values. Except for `none` which is the only custom one we currently use.
Specifiying the architecture is optional, see https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.NETCore.Platforms/src/runtime.json:

```
    win-x64  |  win-x86  |  ...
          \     /
            win
             |
            any
```
```
    linux-x64  |  linux-arm64  |  ...
          \     /
           linux
             |
            any
```
```
    osx-x64  |  osx-arm64  |  ...
          \     /
            osx
             |
            any
```

Reply:
thanks!